core/src/main/java/io/aiven/klaw/service/TopicControllerService.java

```diff
@@ -216,6 +220,25 @@ public ApiResponse createTopicsRequest(TopicRequestModel topicRequestReq) throws
     }
   }
 
+  private void validateAdvancedTopicConfiguration(
+      TopicRequestModel topicRequestModel, TopicRequest topicRequestDao) throws KlawException {
+    Map<String, String> topicConfigs = new HashMap<>();
+    if (null != topicRequestModel.getAdvancedTopicConfigs()) {
+      for (String advancedTopicConfig : topicRequestModel.getAdvancedTopicConfigs()) {
+        String[] strings = advancedTopicConfig.split(":");
+        topicConfigs.put(strings[0], strings[1].trim());
```

Comment:
`TopicRequestModel` comes from outside. It means that there is no guarantee that it contains colon (`:`). It means `strings[1]` could fail with `IndexOUtOfBoundException`